gammapy/image/utils.py

```diff
@@ -260,11 +282,13 @@ def exclusion_distance(exclusion):
 
     Parameters
     ----------
-    TODO
+    exclusion : array_like
```

Comment:
Ok, I just checked in scipy and numpy the corresponding functions that use the input object and they had as a type "array_like" that's why I used it. Fixed.